src/client/sky.cpp

```diff
@@ -358,11 +358,12 @@ void Sky::update(float time_of_day, float time_brightness,
 	}
 
 	m_time_of_day = time_of_day;
+	float raw_time_brightness = 1.0f - 2.0f * abs(time_of_day - 0.5f);
 	m_time_brightness = time_brightness;
 	m_sunlight_seen = sunlight_seen;
 	m_in_clouds = false;
 
-	bool is_dawn = (time_brightness >= 0.20 && time_brightness < 0.35);
+	bool is_dawn = (raw_time_brightness >= 0.395833f && raw_time_brightness < 0.416666f);
```

Comment:
We can also pass both values in from the caller (game.cpp)?
(In that case we might want to rename time_brightness to adjusted_brightness - or something)

Generally there are more places where time_brightness and m_time_brightness are compared with fixed value. Do we need to look at those as well?